modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java

```diff
@@ -154,7 +154,7 @@ protected String getDDMFormFieldValue(
 
 		Stream<DDMFormFieldValue> stream = ddmFormFieldValues.stream();
 
-		return HtmlUtil.render(
+		return HtmlUtil.extractText(
```

Comment:
@linolaoj, have you tested this solution against fields with HTML content, like Paragraph and Rich Text?

These fields use the CKEditor to input rich content, like HTML tags, and maybe the `extractText` can remove valid data from the content inputted on these fields. If this is true, then we can try to remove the break line character only.

Reply:
Hi @natocesarrego, I've tested with HTML content on simple text fields. I'll do some testing with Paragraph and Rich Text.
Thanks!

Reply:
Hi @natocesarrego , on my tests I could not see any difference of behavior for Rich text fields and paragraph other than the removal of the line breaks.
Thanks!

Reply:
Great! Thanks for your tests, @linolaoj. I'll forward your pull. 🙂